samples/getting-started/qrng/Host.cs

```diff
@@ -0,0 +1,41 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using Microsoft.Quantum.Simulation.Core;
+using Microsoft.Quantum.Simulation.Simulators;
+
+
+namespace Qrng{
+    class Driver
+    {
+        static void Main(string[] args)
+        {
+            using (var sim = new QuantumSimulator())
+            {
+                // First we initialize all the variables:
+                var bitString = "0"; // To save the bit string
+                string a = "0"; // Auxiliary string variable
+                int max = 50; // The maximum of the range 
+                bool b = false; // Auxiliary bool
+                int size = Convert.ToInt32(Math.Floor(Math.Log(max, 2.0) + 1));
+                // To calculate the amount of needed bits
+                int output = max + 1; // Int to store the output
+                while(output > max)  // Loop to generate the number
+                {
+                    bitString = "0"; // Restart the bit string if fails 
+                    for(int counter = 0; counter < size ; counter++){
+                        b = (QuantumRandomNumberGenerator.Run(sim).Result == Result.One); 
```

Comment:
Minor, but you could replace this loop with something much closer to what you have on the Python side, which I think could help make it more readable:

```csharp
foreach (var counter in Enumerable.Range(0, size))
{
    bitstring += QuantumRandomNumberGenerator.Run(sim).Result == Result.One ? "1" : "0";
}
```

Using LINQ extension methods, you can make it even simpler:

```csharp
bitString = String.Join("",
    Enumerable.Range(0, size).Select(idx = > 
        QuantumRandomNumberGenerator.Run(sim).Result == Result.One ? "1" : "0"
    )
);
```
That would then look fairly similar to what you currently have at line 26 of the Python side.

Reply:
Ok, I implemented the 2nd version you posted and it works. However, maybe it shows less clearly how to call the Q# operation, which is the main thing we want to show in this tutorial. 
Tell me what you think about this, I can always revert the commits.